src/Symfony/Component/Process/Process.php

```diff
@@ -499,6 +499,54 @@ public function getIncrementalOutput()
     }
 
     /**
+     * Returns an iterator to the output of the process, with the output type as keys (Process::OUT/ERR).
+     *
+     * @param bool $blocking    Whether to use a blocking read call.
+     * @param bool $clearOutput Whether to clear or keep output in memory.
+     *
+     * @throws LogicException in case the output has been disabled
+     * @throws LogicException In case the process is not started
+     *
+     * @return \Generator
+     */
+    public function getIterator($blocking = true, $clearOutput = true)
```

Comment:
This signature does not match the ``\IteratorAggregate``, which looks weird to me.